sumologic/resource_sumologic_cse_outlier_rule.go

```diff
@@ -0,0 +1,236 @@
+package sumologic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"log"
+)
+
+func resourceSumologicCSEOutlierRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicCSEOutlierRuleCreate,
+		Read:   resourceSumologicCSEOutlierRuleRead,
+		Delete: resourceSumologicCSEOutlierRuleDelete,
+		Update: resourceSumologicCSEOutlierRuleUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"aggregate_function": {
+				Type:     schema.TypeMap,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"function": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"arguments": {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+					},
+				},
```

Comment:
```suggestion
				Elem: &schema.Schema{
					"name": {
						Type:     schema.TypeString,
						Required: true,
					},
					"function": {
						Type:     schema.TypeString,
						Required: true,
					},
					"arguments": {
						Type:     schema.TypeList,
						Required: true,
						Elem: &schema.Schema{
							Type: schema.TypeString,
						},
					},
				},
```

You don't need the Resource wrapping the schema like this here. That's causing tflint to fail apparently.